content/html/concepts/tags/terms/abbr/abbr.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '<abbr>'
+Description: 'Indicates an acronym or abbreviation of a longer word or phrase.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'DOM'
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<abbr>` tag indicates an acronym or abbreviation of a longer word or phrase. It can use the `title` attribute which is optional but recommended.
+
+## Syntax
+
+Whenever the `<abbr>` tag is used, it should use the `title` attribute, which indicates the meaning of the abbreviation or acronym.
+
+```html
+<abbr title=""><!-- The acronym or abbreviation here --></abbr>
+```
+
+While using the `title` attribute can be semantically helpful, it is still a good practice in accessibility to spell out the word/phrase in the text the first time before applying the `<abbr>` tag. Some browsers also apply default styling to the `<abbr>` tag like dotted underlines or translating the text into small letters.
+
+## Example
+
+```html
+<p>
+  Tomorrow I'll be heading to the Kennedy Space Center to learn about the
+  National Aeronautics and Space Administration (NASA). Touring through the
+  rockets used in
+  <abbr title="National Aeronautics and Space Administration">NASA</abbr>
+  will be such fun!
+</p>
+```
+
+When the mouse moves over the element, the information written inside the `title` attribute on a tooltip is displayed.
```

Comment:
Wondering if we should have an image for this entry that shows the tooltip displayed over the hovered abbreviation?

Reply:
That's a great idea! Any standards or guidelines that I need to follow for the image?

Reply:
Hope you don't mind me popping in 🍿

Definitely take a look at some existing images on Docs for reference like in the [<q> tag](https://www.codecademy.com/resources/docs/html/tags/q) entry.

The main [Content Standard](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#file-hosting) is to place the image into the media folder.

We can link to the image we place in the media folder like so: https://raw.githubusercontent.com/Codecademy/docs/main/media/<imagename>.<filetype>

Reply:
Great question! Preferably, images for Docs entries should satisfy the following: 

* The should bee a `.jpg` or `.png` file that is less than **1 MB** in size. 
* It has to be stored in the `/media` folder, which is located in the root directory of the `docs` repo.
* To reference the image in the entry markdown files, use the link the starts with `raw.githubcontent...` (which can be obtained with right-click + "Copy Image Address").

Reply:
Thank you so much @yangc95 and @Dusch4593! That was really helpful. I'll be working on that later today.